packages/privacy-toolset/src/cookie-banner/styles.scss

```diff
@@ -0,0 +1,256 @@
+// Granular consent banner (v2).
+$granular-banner-max-width: 1024px;
+
+// Colors.
+$cookie-banner-bg-color: #FFFFFF;
+$cookie-banner-text-color: #000000;
+$cookie-banner-text-muted-color: #444444;
+$cookie-banner-link-color: #117ac9;
+
+// Buttons
+$cookie-banner-main-button-color: $cookie-banner-link-color;
+$cookie-banner-main-button-font-color: #FFFFFF;
+
+// Checkboxes
+$cookie-banner-checkbox-color: $cookie-banner-link-color;
+$cookie-banner-checkbox-bg-color: rgba(196, 195, 195, 0.55);
+
+// Fonts
+$cookie-banner-font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen-Sans", "Ubuntu", "Cantarell",
+"Helvetica Neue", sans-serif;
+
+/**
+ Cookie Banner styling.
+*/
+.cookie-banner {
+	@keyframes fadeIn {
+		0% { opacity: 0; }
+		100% { opacity: 1; }
+	}
+
+	font-family: $cookie-banner-font-family;
+	display: flex;
+	position: fixed;
+	top: auto;
+	left: 50%;
+	transform: translate(-50%);
+	max-width: $granular-banner-max-width;
+	width:100%;
+	margin: 25px;
+	bottom: 25px;
+	margin: 0;
+	max-height: 90vh;
+	z-index: 50001;
+	padding: 20px;
+	background-color: $cookie-banner-bg-color;
+	box-shadow: 0px 8px 16px rgba(0, 0, 0, 0.2), 0px 1px 3px rgba(0, 0, 0, 0.15), 0px 1px 0px rgba(0, 0, 0, 0.05);
+	border-radius: 6px;
+	animation: fadeIn 0.6s;
+	color: $cookie-banner-text-color;
+
+	* {
+		box-sizing: border-box;
+		margin: 0;
+		padding: 0;
+	}
+
+	p {
+		font-size: 15px;
+		font-family: $cookie-banner-font-family;
+		font-weight: normal;
+		line-height: 1.25;
+	}
+	p:not(:first-child) {
+		margin: 18px 0 0 0;
+	}
+	a {
+		color: $cookie-banner-link-color;
+		font-weight: normal;
+		text-decoration: underline;
+	}
+	a:hover,
+	a:active,
+	a:focus {
+		text-decoration:none;
+	}
+
+	.cookie-banner__simple-text-description {
+		width: 75%;
+		letter-spacing: 0.24px;
+		color: $cookie-banner-text-muted-color;
+		font-weight: 400;
```

Comment:
I don't think this is needed as it is the default font-weight for <p> tags

Reply:
Probably not needed. This is also some remnants from the WPcom CSS. I tried to set all the CSS as explicit as possible, since this will run across multiple properties (Calypso, WPcom, Jetpack.com, etc.). To make sure that the styles are specific enough to not be affected by any CSS set in the property itself. We had some issues in Calypso, and also across WPcom blogs, where the parent page styles would affect the cookie banner fonts/alignments/colors.

Reply:
I'll take care of this during the style cleanup.

Reply:
Makes sense to have it if there is concern of being overridden by priority elsewhere. If that is a legitimate concern I see no issue with it 😄